src/arvgvfakecamera.h

```diff
@@ -57,6 +57,12 @@ ArvGvFakeCamera *		arv_gv_fake_camera_new		(const char *interface_name);
 gboolean 			arv_gv_fake_camera_start	(ArvGvFakeCamera *gv_fake_camera);
 void	 			arv_gv_fake_camera_stop		(ArvGvFakeCamera *gv_fake_camera);
 
+/** 
+    arv_gv_fake_camera_get_camera
+    Returns: (transfer none): the ArvFakeCamera instance of this ArvGvFakeCamera object
```

Comment:
API documentation should go into c files

Reply:
That genuinely seems arse about. .h files are the interface files, and that's where you'd naturally go to to see documentation on how to use stuff, as opposed to documentation about implementation detail.
But hey, it's your project.

Reply:
It's better if the documentation is near the implementation. Make it easier to not forget to change it when your working on the code. Also, a change in a c file only triggers the recompilation of one file, while a modification of a header file may cause the recompilation of the whole project.

Reply:
When working on an implementation, one shouldn't be changing the semantics of the interface. Some might go as far as saying one should right the documentation before implementing it - I'm not so extreme, of course. 

More to the point, code is read more often than written - and consulting the interface file is where you go for advice, not the implementation file. I actually hadn't noticed the documentation much at all until you mentioned it.

Re compilation times - its been years since I've worked on a project where compilation time was an issue. I don't think Aravis is in any danger of having this problem!

:)